### PR TITLE
docs: usage

### PR DESCRIPTION
**Pull request by revolunet**
explain the form submission use case

### CODE REVIEW

**Review by matmut7 — APPROVED** [review]

File: docs/docs/use-cases/forms.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Form submissions
+
+e2esdk provide dedicated features for forms submissions and sharing. The full-stack example demonstrates such a scenario
```
Comment (franky47) [inline docs/docs/use-cases/forms.md]:
```suggestion
e2esdk provides dedicated features for form submissions and sharing. The full-stack example demonstrates such a scenario.
```

File: docs/docs/use-cases/forms.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Cover these scenarios
+
+- Random users can submit encrypted data and files
+- The submitted data can be re-edited by submitter
```
Comment (franky47) [inline docs/docs/use-cases/forms.md]:
```suggestion
- The submitted data can be re-edited by the submitter
```

File: docs/docs/use-cases/forms.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- Recipients can invite other recipients to read and interact on the data
+
+## Architecture
+
+While your web application is responsible of storing and distributing the encrypted data and files and provide the correct UI to users, the storing and distribution of decryption keys is entirely handled by e2esdk itself and only happens client-side.
```
Comment (franky47) [inline docs/docs/use-cases/forms.md]:
```suggestion
While your web application is responsible for storing and distributing the encrypted data and files and provide the correct UI to users, the storing and distribution of decryption keys is entirely handled by e2esdk itself and only happens client-side.
```

File: docs/docs/use-cases/forms.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Random users can POST encrypted data
```
Comment (franky47) [inline docs/docs/use-cases/forms.md]:
```suggestion
### Anonymous users can POST encrypted data
```

**suggestion**: Make it clear that users POSTing data **may or may not** be e2esdk-authenticated users.

**Review by franky47 — CHANGES_REQUESTED** [review]
**suggestion**: Minor edits and precisions to be added to distinguish e2esdk-authenticated users from users providing encrypted form data (different auth mechanisms).

**Comment by revolunet** [comment]
@SocketSecurity ignore core-js@3.29.1
@SocketSecurity ignore core-js-pure@3.29.1

**Pull request merged**